data/json/items/comestibles/med.json

```diff
@@ -1155,7 +1155,7 @@
     "name": "joint",
     "description": "Marijuana, cannabis, pot.  Whatever you want to call it, it's rolled up in a piece of paper and ready for smokin'.",
     "weight": "1 g",
-    "volume": "250 ml",
+    "volume": "4 ml",
```

Comment:
The volume value is supposed to be the volume of `stack_size` charges while weight is the mass of a single charge. So 4ml for 100g of joint sounds too little to me.

Reply:
Right, thanks for the reminder.